### ISSUE TITLE
Google Chrome and Mozilla Firefox URLs

### ISSUE THREAD

**Issue by x0tester0x** [rzander/ruckzuck#228]
### Google Chrome
https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise64.msi

### Mozilla Firefox
https://download.mozilla.org/?product=firefox-msi-latest-ssl&os=win64&lang=en-GB
https://download.mozilla.org/?product=firefox-msi-latest-ssl&os=win64&lang=en-US
https://download.mozilla.org/?product=firefox-msi-latest-ssl&os=win64&lang=de
https://download.mozilla.org/?product=firefox-msi-latest-ssl&os=win64&lang=it

**Comment by rzander** [comment]
Google chrome URL is updated...

**Comment by rzander** [comment]
On Firefox, it needs some logic to detect a valid language.
Instead of parsing the available languages online, maybe a fixed list is sufficient.. 
E.g. 
```PowerShell
$lang = "ach,af,sq,ar,an,hy-AM,as,ast,az,eu,be,bn-BD,bn-IN,bs,br,bg,ca,zh-CN,zh-TW,hr,cs,da,nl,en-US,en-GB,en-ZA,eo,et,fi,fr,fy-NL,ff,gd,gl,de,el,gu-IN,he,hi-IN,hu,is,id,ga-IE,it,kn,kk,km,ko,lv,lij,lt,dsb,mk,mai,ms,ml,mr,nb-NO,nn-NO,or,fa,pl,pt-BR,pt-PT,pa-IN,ro,rm,ru,sr,si,sk,sl,son,es-AR,es-CL,es-MX,es-ES,sv-SE,ta,te,th,tr,uk,hsb,uz,vi,cy,xh"  -split(",")
if($lang -contains (Get-Culture).Name) { $lang = (Get-Culture).Name } elseif($lang -contains (Get-Culture).TwoLetterISOLanguageName) { $lang = (Get-Culture).TwoLetterISOLanguageName } else { $lang = "en-US" }
$lang
``` 
What do you think ?

**Comment by x0tester0x** [comment]
Yes, that would be great.

**Comment by rzander** [comment]
implemented in FireFox 14.0.1
